src/Cake.Core/Constants.cs

```diff
@@ -19,11 +19,17 @@ public static class Settings
             public const string ShowProcessCommandLine = "Settings_ShowProcessCommandLine";
         }
 
+        public static class Cache
```

Comment:
As Cache is a part of Cake, not Cake.Core, these constants should be moved there.

Reply:
Moved to Cake/Constants.cs